vss-extension.json

```diff
@@ -25,7 +25,7 @@
         "Azure DevOps Extensions",
         "Marketplace"
     ],
-    "public": true,
+    "public": false,
```

Comment:
Why is this private?

Reply:
To be tested, otherwise we need to add another Repo to facilitate the testing